### PR TITLE
[ru] update date to mention KubeCon NA 2022

### PR DESCRIPTION
**Pull request by Arhell**
en PR https://github.com/kubernetes/website/pull/30722/files
/assign @Potapy4

### CODE REVIEW

File: content/ru/_index.html [hunk 1]
```diff
@@ -41,12 +41,12 @@ <h2>О сложности миграции 150+ микросервисов в Ku
...
         <br>
         <br>
-        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccncna21" button id="desktopKCButton">Посетите KubeCon в Северной Америке, 11-15 октября 2021 года</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe-2022/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccnceu22" button id="desktopKCButton">Посетите KubeCon в Европе, 17-20 мая 2022 года</a>
         <br>
         <br>
         <br>
         <br>
-        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe-2022/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccnceu22" button id="desktopKCButton" button id="desktopKCButton">Посетите KubeCon в Европе, 17-20 мая 2022 года</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccncna21" button id="desktopKCButton" button id="desktopKCButton">Посетите KubeCon в Северной Америке, 24-28 октября 2022 года</a>
```
Comment (sftim) [inline content/ru/_index.html:49]: `kccncna21` doesn't look right. Is it wrong in upstream (English)?

Comment (sftim) [reply content/ru/_index.html:49]: I think it _is_ wrong in upstream. This is therefore correct in terms of localization.
/lgtm

Comment (Arhell) [reply content/ru/_index.html:49]: ok

**Pull request merged**